stat/distuv/weibull_test.go

```diff
@@ -247,6 +248,13 @@ func testWeibull(t *testing.T, dist Weibull, i int) {
 	checkSkewness(t, i, x, dist, tol)
 	checkMedian(t, i, x, dist, tol)
 	checkQuantileCDFSurvival(t, i, x, dist, tol)
-	checkProbContinuous(t, i, x, dist, 1e-10)
+	var probTol float64
```

Comment:
It's worth having a comment to explain why this is necessary.

Reply:
Done.